docs/how-to/extensions/about-extensions.md

```diff
@@ -5,14 +5,16 @@
     They can add many useful things to your forum that aren't avaliabe in the default Flarum software.
 
     Here at FreeFlarum, extensions are treated a little differently. For example, if you want your extension
-    to be added, you have to request it [here](https://www.github.com/gwillem/freeflarum.com/issues/new/choose/).
+    to be added, you have to request it [here][request-your-extensions-here].
+
+    [request-your-extensions-here]: https://github.com/gwillem/freeflarum.com/issues/new?assignees=&labels=%F0%9F%99%8B+Extension%2Ffeature+request&template=extension-request.md&title=%5BInsert+name+of+the+extension%2Ffeature+here%5D
```

Comment:
This feels unnecessary to me, and it will open the edit modal right away which might confuse users. I'd suggest to keep the old version, as that will not feel forced and the user can choose between issues. I'm thinking of just taking them to the issues page, so they can do a research to see if the extension wasn't requested already.

Reply:
> This feels unnecessary to me, and it will open the edit modal right away which might confuse users. I'd suggest to keep the old version, as that will not feel forced and the user can choose between issues. I'm thinking of just taking them to the issues page, so they can do a research to see if the extension wasn't requested already.

@CWKevo Okay, I'll revert it soon.